### PR TITLE
Change error handling

### PR DESCRIPTION
**Pull request by moolitayer**
- Catch only standard errors
- Include original error message in raise error

### CODE REVIEW

File: lib/prometheus/api_client/client.rb [hunk 1]
```diff
@@ -102,8 +102,8 @@ def run_command(command, options)
         response = get(command, options)
 
         JSON.parse(response.body)['data']
-      rescue
-        raise RequestError, 'Bad response from server'
+      rescue StandardError => err
```
Comment (moolitayer) [inline lib/prometheus/api_client/client.rb:105]: We don't want to catch something like `SystemExit`, `NoMemoryError` etc.

**Comment by moolitayer** [comment]
To have really perfect error handling we would need different error classes (ManageIQ for example distinguishes between `MiqException::MiqUnreachableError` and `MiqException::MiqInvalidCredentialsError`)

**Review by grobie — APPROVED** [review]
Thanks!

**Review by yaacov — APPROVED** [review]

**Comment by yaacov** [comment]
will merge when green

**Pull request merged**